boards/arm/s32z270dc2_r52/s32z270dc2_rtu0_r52.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: s32z270dc2_rtu0_r52
+name: NXP X-S32Z270-DC (DC2) on RTU0 Cortex-R52 cores
+type: mcu
+arch: arm
+ram: 1024
+flash: 1024 # to allow Twister to run tests from SRAM
```

Comment:
?

Reply:
I've removed this line. It came from an older of version of Twister we were using internally, but there is not need for it anymore (it was originally to bypass the filter of minimum flash required of some tests since we are running entirely from SRAM).